tests/test-team/template-mgmt-component-tests/template-mgmt-copy-page.component.ts

```diff
@@ -0,0 +1,157 @@
+import { test, expect } from '@playwright/test';
```

Comment:
Do we want to include tests that the new templates look as expected?

Reply:
updated